src/Tizen.Multimedia.MediaPlayer/Player/Player.ErrorHandler.cs

```diff
@@ -15,73 +15,24 @@
  */
 
 using System;
-using System.Collections.Generic;
 using System.ComponentModel;
 
 namespace Tizen.Multimedia
 {
     public partial class Player
     {
-        private static List<Action<Player, int, string>> _errorHandlers;
+        private Action<int, string> _errorHandler;
 
-        private static object _errorHandlerLock = new object();
-
-        /// <summary>
-        /// This method supports the product infrastructure and is not intended to be used directly from application code.
-        /// </summary>
-        /// <since_tizen> 4 </since_tizen>
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        protected static void AddErrorHandler(Action<Player, int, string> errorHandler)
```

Comment:
This API can be removed without ACR because it is INTERNAL API. However, ABI is broken, so be very careful. 
Please make sure that you do not use this API.